### PR TITLE
Feature/support ts schema

### PR DESCRIPTION
**Pull request by maticzav**
This PR adds support for importing the schema from typescript files.

### CODE REVIEW

File: packages/graphqlgen/tests/validation/validateModels.test.ts [hunk 1]
```diff
@@ -74,4 +74,49 @@ describe('test validateModels()', () => {
...
+          override: {
+            Post: relative('./mocks/tsSchemaConst/model.ts:PostModel'),
+          },
+        },
+      },
+      true,
+    )
+  })
+
+  test('import schema gql default', () => {
```
Comment (jasonkuhrt) [inline packages/graphqlgen/tests/validation/validateModels.test.ts]:
```suggestion
  test('import schema gql const', () => {
```

**Comment by jasonkuhrt** [comment]
One reason this is important, as stated by @maticzav in slack:

> This is especially important for deployments using `now` because .graphql file cannot be imported - the deployment ignores it when preparing the bundle.

**Comment by steida** [comment]
@jasonkuhrt @maticzav 

Can you please describe the use case in more detail? I haven't a problem with it. Thank you.

**Comment by jasonkuhrt** [comment]
@steida I'll let @maticzav expand on the `now` one as I'm not a zeit user myself.

However users should be able to import from TypeScript files anyways. There are quite a few examples by and of Apollo servers using the ``const typeDefs = gql`...` `` so having this start to work is good a good step. I realize that a complex project doing modular ``const typeDefs = gql`...` `` will not work yet. I think we could achieve that with a scan across the user's project. But more complicated. Lets see if users start to ask for it. Other priorities for now.

**Comment by maticzav** [comment]
Hey @steida 👋,

So, the reason why this is important is that firstly, Apollo users to my understanding had to import and parse their schema during the runtime, and secondly, deployed applications couldn't work with Now 2.0. The reason behind this is that Now uses `ncc` to compile lambdas out of source code, therefore, they wouldn't bundle schema because it was a runtime dependency.

I hope this gives you a good enough idea 🙂

**Comment by steida** [comment]
@maticzav Thank you. Btw, deployed API works for me here in https://github.com/este/este

**Pull request merged**